### ISSUE TITLE
Hi!

### ISSUE THREAD

**Issue by scj0709** [open-mmlab/mmediting#448]
Hi!
I am really impressed with GLEAN.
I want to train, but I can't do well due to the error below.
![image](https://user-images.githubusercontent.com/40060713/126759452-eb39a510-01fc-42ff-9605-0438fad994b6.png)
Help me!

**Comment by ckkelvinchan** [comment]
It seems that your `mmcv` is not installed correctly.

**Comment by scj0709** [comment]
pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu100/torch1.6.0/index.html

I installed mmcv as instructed above. I don't know what the problem is.

**Comment by innerlee** [comment]
Mmcv is trying to make the installation easier, please seek for help at mmcv repo for its installation

**Comment by ckkelvinchan** [comment]
> pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu100/torch1.6.0/index.html
> 
> I installed mmcv as instructed above. I don't know what the problem is.

Please try the installation command in the official [mmcv](https://github.com/open-mmlab/mmcv) repo. Do remember to select the correct CUDA version and PyTorch version.

**Comment by scj0709** [comment]
Thank you very much for your reply.
So, when I reinstalled it according to the Pytorch and CUDA version, the following error appears.
![image](https://user-images.githubusercontent.com/40060713/126770057-051de532-ab58-4612-a11c-b68ec2f06320.png)

**Comment by ckkelvinchan** [comment]
Are you using PyTorch 1.4 or 1.5? Could you try installing a newer version of PyTorch and reinstall mmcv?

**Comment by scj0709** [comment]
Thank you for answer.
I'm getting the same error now on both Pytorch 1.4 and 1.6.
My CUDA version is 10.0. Could this be the cause?

**Comment by scj0709** [comment]
It was resolved with good advice. :)
But i ran into another problem.
![image](https://user-images.githubusercontent.com/40060713/126923878-00e9d09f-a059-44f4-bb50-510382856402.png)

**Comment by scj0709** [comment]
Thank you for answer.
I solved it!
![image](https://user-images.githubusercontent.com/40060713/126930205-9263643c-2b25-4d6b-ad93-78ceab55da85.png)
I ran into this problem.
Do I have to degrade the FFHQ dataset myself and put it in??

**Comment by ckkelvinchan** [comment]
> Thank you for answer.
> I solved it!
> ![image](https://user-images.githubusercontent.com/40060713/126930205-9263643c-2b25-4d6b-ad93-78ceab55da85.png)
> I ran into this problem.
> Do I have to degrade the FFHQ dataset myself and put it in??

Yes. We use MATLAB `imresize` to downsample the images.

**Comment by scj0709** [comment]
I am so glad for your kind reply!
It was confirmed that it works well.
But I don't know how to save the image that has been validated in the middle.
I changed save_image = True in the config file, but nothing is saved.

**Comment by scj0709** [comment]
I succeeded in training.
I am so happy.
Could you tell me where is the reverse loss or loss calculation in the code?
I saw what loss was used in the configuration file.
But I want to know where is the code where the actual output comes out and the gt and loss are calculated.
Thank you very much!!

**Comment by ckkelvinchan** [comment]
> I succeeded in training.
> I am so happy.
> Could you tell me where is the reverse loss or loss calculation in the code?
> I saw what loss was used in the configuration file.
> But I want to know where is the code where the actual output comes out and the gt and loss are calculated.
> Thank you very much!!

Hello, the training of GLEAN follows the class `SRGAN`, you can check [here](https://github.com/open-mmlab/mmediting/blob/642b759316c6f85de9deaeb89a9790c117d6f55a/mmedit/models/restorers/srgan.py#L10) fore the training details.

**Comment by scj0709** [comment]
Thank you very much for your reply.
you are so kind!!

**Comment by scj0709** [comment]
Hello!
May I know the hyperparameters used to perform 64x SR in GLEAN?

**Comment by ckkelvinchan** [comment]
> Hello!
> May I know the hyperparameters used to perform 64x SR in GLEAN?
> I am training 64x SR using the celeb-HQ dataset as a training set, and I am training the 85,000th, but the results are not good.

The hyperparameters are the same.
